bigtable_automation/README.md

```diff
@@ -2,32 +2,38 @@
 
 BT automation works as follows:
 
-1.  The Prophet Generator Borg job notifies a
-    [Cloud Function](https://pantheon.corp.google.com/functions/details/us-central1/prophet-cache-trigger?project=datcom-store&tab=general)
-    (GCF)
-    which creates a new BT table, scales up BT resources (for base cache builds
-    only) and kicks off the
-    [CSV-to-BT Dataflow job](https://pantheon.corp.google.com/dataflow/jobs?organizationId=433637338589&project=datcom-store)
-    stored in
-    [datcom-dataflow-templates](https://pantheon.corp.google.com/storage/browser/_details/datcom-templates/templates/csv_to_bt).
+1. The Prophet Generator Borg job notifies a
+   [Cloud Function](https://pantheon.corp.google.com/functions/details/us-central1/prophet-cache-trigger?project=datcom-store&tab=general)
+   (GCF)
+   which creates a new BT table, scales up BT resources (for base cache builds
+   only) and kicks off the
+   [CSV-to-BT Dataflow job](https://pantheon.corp.google.com/dataflow/jobs?organizationId=433637338589&project=datcom-store)
+   stored in
+   [datcom-dataflow-templates](https://pantheon.corp.google.com/storage/browser/_details/datcom-templates/templates/csv_to_bt).
 
-2.  When the Dataflow job completes, it notifies the same cloud function to
-    scale down BT resources (for base cache builds only).
+2. When the Dataflow job completes, it notifies the same cloud function to
+   scale down BT resources (for base cache builds only).
 
-3.  The notifications between Borg-to-GCF and Dataflow-to-GCF happen using
-    per-version state files located in GCS:
+3. The notifications between Borg-to-GCF and Dataflow-to-GCF happen using
+   per-version state files located in GCS:
 
-    *   Base cache:
-        [/bigstore/datcom-control/base/{BT_Table}/...](https://pantheon.corp.google.com/storage/browser/datcom-control/base)
-    *   Branch cache:
-        [/bigstore/datcom-control/branch/{BT_Table}/...](https://pantheon.corp.google.com/storage/browser/datcom-control/branch)
+   - Base cache:
+     [/bigstore/datcom-control/base/{BT_Table}/...](https://pantheon.corp.google.com/storage/browser/datcom-control/base)
+   - Branch cache:
+     [/bigstore/datcom-control/branch/{BT_Table}/...](https://pantheon.corp.google.com/storage/browser/datcom-control/branch)
```

Comment:
Don't the paths change across private vs. prod?

Reply:
Added path for private